purchase_default_terms_conditions/models/purchase.py

```diff
@@ -10,7 +10,10 @@ class PurchaseOrder(models.Model):
 
     @api.onchange("partner_id", "company_id")
     def onchange_partner_id(self):
-        if self.partner_id.purchase_note:
+        if (
+            self.partner_id.purchase_note
+            and self.partner_id.purchase_note != "<p><br/></p>"
```

Comment:
I think you can try "is_html_empty"  by `from odoo.tools import is_html_empty` to check the purchase_note is empty or not.

Reply:
Done @quannguyen1560.. Can you check?